config.yaml

```diff
@@ -0,0 +1,11 @@
+options:
+  cuda-version:
+    type: string
+    default: "8.0.61"
+    description: |
+      The version of CUDA to be intalled.
+  cuda-subversion:
```

Comment:
CUDA_VERSION without the subversion is used also in install_nvidia_gdk (eg https://github.com/juju-solutions/layer-nvidia-cuda/blob/master/reactive/cuda.sh#L61) but I think this is dead code.